fe/fe-core/src/main/java/org/apache/doris/qe/ConnectContext.java

```diff
@@ -640,12 +640,13 @@ public String getCurrentConnectedFEIp() {
         return currentConnectedFEIp;
     }
 
-    public void resetExecTimeout() {
-        if (executor != null && executor.isInsertStmt()) {
-            // particular timeout for insert stmt, we can make other particular timeout in the same way.
-            // set the execution timeout as max(insert_timeout,query_timeout) to be compatible with older versions
-            executionTimeoutS = Math.max(sessionVariable.getInsertTimeoutS(), executionTimeoutS);
-        }
+    public void setExecTimeout(int timeout) {
+        executionTimeoutS = timeout;
+    }
+
+    public long resetExecTimeoutByInsert() {
```

Comment:
@TangSiyang2001 I think this is more clean to understand since now it only used by insert. But I keep `setExecTimeout` for future use

Reply:
OK, I aggre with you.